app/templates/DonationForm.html.twig

```diff
@@ -175,21 +183,54 @@
                             email: $( '.confirm-email' )
                         } ),
                         stateKey: 'donationFormContent'
+                    },
+                    {
+                        viewHandler: WMDE.View.createBankDataDisplayHandler(
+                            $( '.confirm-iban' ),
+                            $( '.confirm-bic' ),
+                            $( '.confirm-bank-name' )
+                        ),
+                        stateKey: 'donationFormContent'
                     }
                 ],
                 store
         );
 
         // connect DOM elements to actions
 
-        $( '#continueFormSubmit1,#continueFormSubmit2' ).click( function () {
-            // TODO check if page is valid
-            store.dispatch( actions.newNextPageAction() );
+        $( '#continueFormSubmit1' ).click( function () {
+            if ( store.getState().validity.amount ) {
+                store.dispatch( actions.newNextPageAction() );
+            } else {
+                alert( 'Bitte füllen Sie das Formular komplett aus.' );
```

Comment:
In my opinion, calling `alert()` is not so nice. Do you plan to replace this later?